packages/htmlbars-compiler/tests/fragment-test.js

```diff
@@ -178,13 +183,13 @@ test('test auto insertion of text nodes for needed edges a fragment with morph m
 
   var t = morphs[0].start;
   equal(t.nodeType, 3);
-  equal(t.textContent , '');
+  equal(t.textContent !== undefined ? t.textContent : t.toString() , '');
   equal(morphs[1].start, null);
   equal(morphs[1].end, null);
 
   equal(morphs[2].start, morphs[1].parent());
   equal(morphs[2].end.nodeType, 3);
-  equal(morphs[2].end.textContent, '');
+  equal(morphs[2].end.textContent  !== undefined ? morphs[2].end.textContent : morphs[2].end.toString(), '');
```

Comment:
These assertions with ternary statements are really confusing. They should be the same for all browsers.